### PR TITLE
fix: disabled JoyID connector

### PR DESCRIPTION
**Pull request by ahonn**
See https://github.com/sporeprotocol/spore-demo/issues/67 and https://github.com/sporeprotocol/spore-demo/issues/69

### CODE REVIEW

**Comment by ShookLyngs** [comment]
H, I have a question: why not remove it from the `JoyIdConnector ` from the `connectors`?
https://github.com/sporeprotocol/spore-demo/blob/0d95ee86662be316fa80970360d01a3f59106823/src/pages/_app.tsx#L95-L98

**Comment by ahonn** [comment]
> Hi, I have a question: why not remove it from the `JoyIdConnector ` from the `connectors`?
> 
> https://github.com/sporeprotocol/spore-demo/blob/0d95ee86662be316fa80970360d01a3f59106823/src/pages/_app.tsx#L95-L98

For users who have previously used JoyID connect, removing the JoyId connector directly may result in errors and unexpected inability to control Spore that was previously Minted.
Therefore, I am more inclined to only hide it for new users without affecting existing users who use JoyID for connect.

**Comment by ShookLyngs** [comment]
> For users who have previously used JoyID connect, removing the JoyId connector directly may result in errors and unexpected inability to control Spore that was previously Minted. Therefore, I am more inclined to only hide it for new users without affecting existing users who use JoyID for connect.

Okay, now I understand. This is a simple change to remove the JoyID option from the wallet connect modal without affecting other users who have already connected via JoyID. However, if the connected user accidentally clicks the "disconnect" button, does it mean that the user will lose the ability to reconnect?

For that, I recommend marking the connector as somewhat "deprecated" and still showing it in the wallet connect modal, but lowering its ordering. Additionally, add some text to explain why this option is currently not recommended. Do you think it would be a better approch for everyone?

**Comment by ahonn** [comment]
> > For users who have previously used JoyID connect, removing the JoyId connector directly may result in errors and unexpected inability to control Spore that was previously Minted. Therefore, I am more inclined to only hide it for new users without affecting existing users who use JoyID for connect.
> 
> Okay, now I understand. This is a simple change to remove the JoyID option from the wallet connect modal without affecting other users who have already connected via JoyID. However, if the connected user accidentally clicks the "disconnect" button, does it mean that the user will lose the ability to reconnect via JoyID thereafter?
> 
> For that, I recommend marking the connector as somewhat "deprecated" and still showing it in the wallet connect modal, but lowering its ordering. Additionally, add some text to explain why this option is currently not recommended. Do you think it would be a better approch for everyone?

There are currently only two connection methods available, so simply marking it as deprecated may not be sufficient. 
My idea is to discourage future users from using the JoyID connector. The current changes has minimal impact. 

I believe it is acceptable that existing users who disconnect from JoyID connect will no longer be able to use it, as the purpose of this change is to prevent any confusion for them. In the README, I have also clearly stated that we have temporarily hidden the integration of JoyID.

Perhaps we can consider show tips when existing users disconnect from JoyID connect, but I believe the necessity of doing so is relatively low. What do you think?

**Comment by ShookLyngs** [comment]
> Perhaps we can consider show tips when existing users disconnect from JoyID connect, but I believe the necessity of doing so is relatively low. What do you think?

From my perspective, I believe it would be more reasonable to add a card/box in the ConnectModal to explain why the JoyID option is no longer visible. This approach ensures that users understand the change and discourages them from attempting to connect using JoyID again, not when they're trying to disconnect.

**Comment by ahonn** [comment]
> > Perhaps we can consider show tips when existing users disconnect from JoyID connect, but I believe the necessity of doing so is relatively low. What do you think?
> 
> From my perspective, I believe it would be more reasonable to add a card/box in the ConnectModal to explain why the JoyID option is no longer visible. This approach ensures that users understand the change and discourages them from attempting to connect using JoyID again, not when they're trying to disconnect.

Cool, I think adding a message in ConnectModal to provide an explanation is better.

**Comment by ahonn** [comment]
@ShookLyngs Now we can still see JoyID in ConnectModal, but it is marked as disabled. We will display a message about JoyID being deprecated and link to https://github.com/sporeprotocol/spore-demo/issues/69

**Comment by ShookLyngs** [comment]
> @ShookLyngs Now we can still see JoyID in ConnectModal, but it is marked as disabled. We will display a message about JoyID being deprecated and link to #69

Cool, I have a question: do you think it's a good idea to change the ordering of the elements in the modal? I feel like the current ordering (Message -> MetaMask -> JoyID) might hinder MetaMask-only users as they only want to connect via MetaMask. 

Would it be better if we change the order to the following:

```
1. MetaMask
2. JoyID
3. Message
```

File: src/components/ConnectModal.tsx [hunk 1]
```diff
@@ -49,13 +50,29 @@ export default function ConnectModal(props: ConnectModalProps) {
 
   return (
     <Flex direction="column" gap="md">
+      <Alert variant="filled" color="brand.1" title="JoyID deprecated">
+        We are deprecate the JoyID (Omnilock) option because it is more recommended to connect to
```
Comment (ShookLyngs) [inline src/components/ConnectModal.tsx:54]: "We are deprecating" or "We've deprecated"?

**Review by ShookLyngs — CHANGES_REQUESTED** [review]

**Comment by ahonn** [comment]
> > @ShookLyngs Now we can still see JoyID in ConnectModal, but it is marked as disabled. We will display a message about JoyID being deprecated and link to #69
> 
> Cool, I have a question: do you think it's a good idea to change the ordering of the elements in the modal? I feel like the current ordering (Message -> MetaMask -> JoyID) might hinder MetaMask-only users as they only want to connect via MetaMask.
> 
> Would it be better if we change the order to the following:
> 
> ```
> 1. MetaMask
> 2. JoyID
> 3. Message
> ```

Usually, announcements or notifications should be placed at the top, so I have followed this convention.
I don’t have a particular preference for where to place it, I think both options are fine.

**Comment by ShookLyngs** [comment]
> Usually, announcements or notifications should be placed at the top, so I have followed this convention. I don’t have a particular preference for where to place it, I think both options are fine.

Yeah, I think if you wish the message to be eye-catching, the current ordering is just corret.
I have no other questions for it, thanks.

**Review by ShookLyngs — APPROVED** [review]

**Pull request merged**